### DISCUSSION TITLE
Can't access Github Pro Student benefits

### DISCUSSION THREAD

**Discussion by PaulMagos** [community/community#66300] [GitHub Education]
### Select Topic Area

Question

### Body

 I have applied to the global campus and had the benefits for around one month, and then they just stopped my benefits. Other colleagues at my Uni (University of Pisa) have the benefits and all works correctly. Others like me don't have any benefits and if we try to apply it says that we already have the coupon applied to our account.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @PaulMagos ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!
